src/rpdk/core/init.py

```diff
@@ -172,7 +196,52 @@ def setup_subparser(subparsers, parents):
     parser.set_defaults(command=ignore_abort(init))
 
     parser.add_argument(
-        "--force", action="store_true", help="Force files to be overwritten."
+        "-f",
+        "--force",
+        action="store_true",
+        help="Force files to be overwritten.",
+    )
+
+    parser.add_argument(
+        "-l",
+        "--language",
+        help="""Select a language for code generation.
+            The language plugin needs to be installed.""",
+    )
+
+    parser.add_argument(
+        "-t",
+        "--type-name",
+        help="Select the name of the resource type.",
+    )
+
+    parser.add_argument(
+        "-d",
+        "--use-docker",
+        action="store_true",
+        help="""Use docker for python platform-independent packaging.
+            This is highly recommended unless you are experienced
+            with cross-platform Python packaging.""",
+    )
+
+    parser.add_argument(
+        "-n",
+        "--namespace",
```

Comment:
@jotompki do you think we could use something like a config file to specify all the metadata - [ConfigArgParse](https://pypi.org/project/ConfigArgParse/) is one of the options